static/css/components/read-panel.less

```diff
@@ -20,6 +20,7 @@
   }
   strong,
   .cta-section-title {
+    font-size: 12.5px;
```

Comment:
Isn't this is redundant? I thought this is identical to the rule just above it 
.cta-section { font-size: .78em? }

Reply:
It was causing the .cta-section-title to be 10px